src/tasks/migrations/0004_auto_20220302_2250.py

```diff
@@ -4,7 +4,6 @@
 
 
 class Migration(migrations.Migration):
-
```

Comment:
All of the formatting changes here are the result of the newly-introduced [2023 stable style, enforced as of Black 23.1.0](https://github.com/psf/black/releases/tag/23.1.0)